lib/logflare/backends/ingest_event_queue/queue_janitor.ex

```diff
@@ -53,20 +52,12 @@ defmodule Logflare.Backends.IngestEventQueue.QueueJanitor do
   def do_drop(state) do
     sid_bid = {state.source_id, state.backend_id}
     # clear out ingested events
-    pending_size =
-      Backends.get_and_cache_local_pending_buffer_len(state.source_id, state.backend_id)
-
-    if pending_size > state.remainder do
-      # drop all ingested
-      IngestEventQueue.truncate(sid_bid, :ingested, 0)
-    else
-      IngestEventQueue.truncate(sid_bid, :ingested, state.remainder)
-    end
-
+    IngestEventQueue.truncate_queues(sid_bid, :ingested, state.remainder)
```

Comment:
would leave a large total remainder across all queues if there are many pipelines.
can drop all ingested if there is sufficient pending count.